travis_configure.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
```

Comment:
Sorry made a comment but somehow didn't post it. I tried with `sh` but some of the syntax is  not compatible. Furthermore `travis-ci` uses bash scripts in their examples in their github repo. So I will stick with `bash`